src/content/it/fundamentals/performance/critical-rendering-path/index.md

```diff
@@ -2,15 +2,15 @@ project_path: /web/_project.yaml
 book_path: /web/fundamentals/_book.yaml
 description: Ottimizzazione del percorso di rendering critico attraverso l'assegnazione di priorità alla visualizzazione del contenuto che è correlato all'azione principale che l'utente vuole effettuare sulla pagina.
 
-{# wf_updated_on: 2014-04-27 #}
+{# wf_updated_on: 2017-07-14 #}
 {# wf_published_on: 2014-03-31 #}
 
 # Percorso di rendering critico {: .page-title }
 
 {% include "web/_shared/contributors/ilyagrigorik.html" %}
 
 
-L'ottimizzazione del percorso di rendering critico è cruciale per il miglioramento delle prestazioni delle nostre pagine: il nostro scopo è assegnare priorità e visualizzare il contenuto relativo all'azione principale che l'utente vuole eseguire sulla pagina.
+_L'ottimizzazione del percorso di rendering critico_ è cruciale per il miglioramento delle prestazioni delle nostre pagine: il nostro scopo è assegnare priorità e visualizzare il contenuto relativo all'azione principale che l'utente vuole eseguire sulla pagina.
 
 La fornitura di un'esperienza di rete veloce richiede capacità elevate da parte del browser. La maggior parte di questo lavoro è nascosto a noi sviluppatori web: scriviamo il markup e la nostra pagina viene visualizzata sullo schermo. In che modo il browser passa dall'utilizzo di HTML, CSS e JavaScript al rendering dei pixel sullo schermo?
 
```

Comment:
Translation of the following segment is missing:
Optimizing for performance is all about understanding what happens in these intermediate steps between receiving the HTML, CSS, and JavaScript bytes and the required processing to turn them into rendered pixels - that's the critical rendering path.

Reply:
If I'm not wrong the translation of that segment is on line 17.

Reply:
Yes, Luca you are correct.